recipes-browser/wpewebkit/wpewebkit_2.38.0.bb

```diff
@@ -0,0 +1,22 @@
+require wpewebkit.inc
+require conf/include/devupstream.inc
```

Comment:
I dont think this include is a good option. There is not guarantees ToT is always builtable.

Reply:
I was trying to mimic what you did when creating 2.36.0, I think, so I moved it to this one from `2.36.7.bb`.  Isn't this "devupstream" supposed to be attached to the .bb of the latest version?

Also I don't know if you suggest to keep it in 2.36 or to drop it from all .bbs.

Reply:
You are right, I am wrong here. For some reason I was thinking in webkit-bleeding.inc not in devupstream.inc